packages/logging-controller/README.md

```diff
@@ -0,0 +1,15 @@
+# `@metamask/logging-controller`
+
+Stores logs about signature request.
```

Comment:
It looks like the description in `package.json` says "Manages a list of logs for signature requests". Maybe this should be changed too? It seems like there'd be no reason to make the description between the README and `package.json` different.

``` suggestion
Manages a list of logs for signature requests.
```

Reply:
Actually we should change the package.json to match this, because its more than just signature requests that can store logs.

Reply:
Oh i see. i hadnt pushed changes I made to the readme yet 😓